### DISCUSSION TITLE
Erro no arquivo binding.dart

### DISCUSSION THREAD

**Discussion by s-alexsan** [Flutterando/forum#5] [Perguntas e Respostas] [answered]
Fala pessoal.

Fiquei um tempo sem utilizar o app e fiz uma nova atualização no flutter e afins e toda vez que executo o projeto aparece o seguinte erro.
![Captura de tela 2022-09-12 073305](https://user-images.githubusercontent.com/64074234/189632829-d3857fa5-0097-4df0-8077-a09b727a9d6c.png)
 A aplicação funciona normalmente, porém estou com dúvida do motivo de acontecer esse erro e como faço para corrigir.

Agradeço pela ajuda!! 😄

**Comment by HudsonAfonso** [comment, marked as answer]
Antes do Flutter 3.x.x `WidgetsBinding.instance` retornava `WidgetsBinding?`
com a nova versão ele não tem a possibilidade de retornar `null` então todo código antigo normalmente tem um `WidgetsBinding.instance!` e ele da um warning nisso...

vc pode tentar upar o `charts_flutter`, quem sabe resolveram isso na package

**Reply by s-alexsan** [reply]
Verifiquei aqui e o o plugin `charts_flutter` está na sua ultima versão utilizando o flutter 2.1.2 então por esse motivo fica aparecendo os warning, certo? Tem de alguma forma corrigir esses erros?

**Reply by edeegg** [reply]
Você pode corrigir manualmente o erro mudando o plugin para null safe , baixe o plugin faça as alterações e depois adicione ao seu projeto.

**Reply by s-alexsan** [reply]
Legal, vou fazer isso então. 
Muito obrigado pela ajuda!!!

**Comment by toshiossada** [comment]
Isso são apenas warnings, não são erros

**Reply by s-alexsan** [reply]
Obrigado @toshiossada, me deixa mais aliviado.
